app/controllers/institutes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InstitutesController < ApplicationController
+    before_filter :load_institute, except: [:index]
```

Comment:
@Luke-kb looks like some of your settings have changed to 4 spaces ?
The standard style for Ruby is 2 spaces
https://github.com/rubocop/ruby-style-guide#indentation

Reply:
As well as a bunch of the files below

Reply:
😮  thanks for catching that